CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
+### 0.7.0 - Sep 22, 2021
+
+* Added support for stream commands in regular API.
+* Added support for authentication using AUTH with username.
+* Added adapters for event libraries libuv, libev and GLib.
+* Improved memory efficiency.
+* Renamed API function `redisClusterSetOptionMaxRedirect(..)`
+  to `redisClusterSetOptionMaxRetry(..)`.
```

Comment:
C functions are usually described with empty parantheses (meaning parameters are unspecified, as opposed to `(void)` which means no parameters).

```suggestion
* Renamed API function `redisClusterSetOptionMaxRedirect()`
  to `redisClusterSetOptionMaxRetry()`.
```